### DISCUSSION TITLE
Why Would WGPU Register Vulkan 1.1.119 and `vulkaninfo` Register 1.2.131?

### DISCUSSION THREAD

**Discussion by zicklag** [gfx-rs/wgpu#1642] [Q&A] [answered]
Not a big deal to me as WGPU is working either way, but I was wondering why `vulkaninfo` and WGPU would detect different versions of vulkan on the same hardware.

I'm mostly just curious and don't care if nobody answers :), but if anybody knows right off or feels like it'd be usefult to look into...

**Comment by kvark** [comment, marked as answer]
Maybe you are on dual-GPU, and you are looking at the *other* adapter in wgpu?

**Reply by zicklag** [reply]
Oh, yeah. That's right. I had mistakenly assumed ( obvious now that I think about it ) that my fancy, powerful GPU would have a later version of vulkan than the Intel one.
